src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx

```diff
@@ -7,6 +7,49 @@ topics:
 
 Use this document to learn how to review and peer edit docs for you fellow writers in GitHub. Check the [Tech Writer workflow doc](/docs/style-guide/writer-workflow/tech-writer-workflow/) for info on how to set up your local environment. To troubleshoot GitHub issues, see [our guide](/docs/style-guide/writer-workflow/github-troubleshooting/).
 
+## What needs a peer review [#peer-reqs]
+
+Any substantial pull request should be peer reviewed. The line between substantial and trivial is a bit blurry, so refer to the following table when in doubt:
```

Comment:
I would bold or otherwise emphasize that first sentence about Any substantial pull request.